test-cases/longevity/longevity-100GB-48h-cloud-CloudLimitedChaosMonkey-tls.yaml

```diff
@@ -15,7 +15,8 @@ post_behavior_db_nodes: "destroy"
 post_behavior_loader_nodes: "destroy"
 post_behavior_monitor_nodes: "destroy"
 
-nemesis_class_name: 'LimitedChaosMonkey'
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_include_filter: ['limited']
```

Comment:
have you tested this `nemesis_include_filter`?

Reply:
Yes, in the previous iteration, see above comment.

Reply:
On this one it's on purpose, because I want it to run randomly for now.